<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_redroad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>红外循迹开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_debug_on">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开启命令调试</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_debug">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>497</width>
      <height>293</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_lround">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>291</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;3ds&quot;;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
       </property>
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_lu">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↖</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_u">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↑</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_ru">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↗</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_l">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_stop">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;
color: rgb(172, 0, 0);</string>
         </property>
         <property name="text">
          <string>■</string>
         </property>
         <property name="shortcut">
          <string>Space</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_right">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_ld">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↙</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_d">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↓</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_rd">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;3ds&quot;;</string>
         </property>
         <property name="text">
          <string>↘</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rround">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>291</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;3ds&quot;;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>271</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_board_1">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>131</height>
        </size>
       </property>
       <property name="text">
        <string>板_升</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_board_2">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>131</height>
        </size>
       </property>
       <property name="text">
        <string>板_降</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>271</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_pin_1">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>131</height>
        </size>
       </property>
       <property name="text">
        <string>针_升</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pin_2">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>131</height>
        </size>
       </property>
       <property name="text">
        <string>针_降</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_scan">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>83</y>
      <width>257</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>257</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>扫描</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_read">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>153</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>118</y>
      <width>257</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>257</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_com">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>11</y>
      <width>259</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>259</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>40</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_baud">
    <property name="geometry">
     <rect>
      <x>774</x>
      <y>40</y>
      <width>205</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>205</width>
      <height>31</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
